layouts/partials/survey.html

```diff
@@ -1,20 +1,24 @@
 <h2>How you can help</h2>
 
-<a class="survey-link clickable-banner-link" href="https://forms.gle/5wFxEz4X5Z8Wohk86">
-  <div class="survey clickable-banner">
-    <div class="clickable-banner-title">
-      🏠 Fill Out Homeowner's Survey
-    </div>
-  </div>
-</a>
+<!--
+
+     <a class="survey-link clickable-banner-link" href="https://forms.gle/5wFxEz4X5Z8Wohk86">
+     <div class="survey clickable-banner">
+     <div class="clickable-banner-title">
+     🏠 Fill Out Homeowner's Survey
+     </div>
+     </div>
+     </a>
+
+-->
 
-<a class="risk-reduction-link clickable-banner-link" href="https://forms.gle/91CGZHuW7Pwp2G5m9">
+<a class="risk-reduction-link clickable-banner-link" href="https://tinyurl.com/2p86s9xz">
   <div class="risk-reduction clickable-banner">
     <div class="clickable-banner-title">
       🍁 Firewise Defensible Space Work Log
     </div>
     <div class="clickable-banner-description">
-      In order to receive and maintain Firewise USA® Recognition status we need to report an annual wildfire risk reduction investment and/or defensible space volunteer hours.  This link will lead you to an online form to report the work or expenses incurred toward these efforts which will help us in the qualification process.  Thank you for taking the time to complete the form.
+      In order to maintain the Firewise USA® Recognition status we need to report an annual wildfire risk reduction investment and/or defensible space volunteer hours.  This link will lead you to an online form to report the work or expenses incurred toward these efforts which will help us in the qualification process.  Thank you for taking the time to complete the form.
```

Comment:
@stefanv I think we're missing a couple of commas here, how about - "In order to maintain the Firewise USA® Recognition status, we need to report an annual wildfire risk reduction investment and/or defensible space volunteer hours. This link will lead you to an online form to report the work or expenses incurred toward these efforts, which will help us in the qualification process. Thank you for taking the time to complete the form." - ?

Want me to push it to this branch?

Reply:
Go for it, thanks!